core-plugins/src/test/java/co/cask/hydrator/plugin/transform/PythonEvaluatorTest.java

```diff
@@ -132,8 +132,8 @@ public void testSimple() throws Exception {
     Assert.assertEquals(expectedListField, output.get("arrayField"));
   }
 
-  @Test
-  public void testSchemaValidation() throws Exception {
+  @Test(expected = Exception.class)
+  public void testSchemaValidationInvalidScript() throws Exception {
```

Comment:
nit: name is misleading, since no schema validation is being tested. Can rename to something like testScriptCompilationValidation